include/fcl/narrowphase/detail/gjk_solver_indep.h

```diff
@@ -50,14 +49,13 @@ namespace detail
 
 /// @brief collision and distance solver based on GJK algorithm implemented in fcl (rewritten the code from the GJK in bullet)
 template <typename S_>
-struct GJKSolver_indep
+struct FCL_EXPORT GJKSolver_indep
 {
   using S = S_;
 
   /// @brief intersection checking between two shapes
   /// @deprecated use shapeIntersect(const Shape1&, const Transform3<S>&, const Shape2&, const Transform3<S>&, std::vector<ContactPoint<S>>*) const
   template<typename Shape1, typename Shape2>
-  FCL_DEPRECATED
```

Comment:
Is this an oversight or removed intentionally? `fcl_export.h` still has the definition of `FCL_DEPRECATED`. This deprecation was introduced in #52 for backward compatibility. If we want to remove this deprecation, I would like to remove this function entirely not to cause confusion of which `shapeIntersect` should be used.